assets/airports/lipz.json

```diff
@@ -0,0 +1,2329 @@
+            ["45.501568314762565", "12.342056035995483", "45.50150063714179",  "12.34215259552002 "],
+            ["45.50150063714179",  "12.34215259552002 ", "45.501504397011736", "12.342297434806824"],
+            ["45.501504397011736", "12.342297434806824", "45.50153823583003",  "12.34340786933899 "],
+            ["45.50153823583003",  "12.34340786933899 ", "45.501572074628",    "12.34361708164215 "],
+            ["45.501572074628",    "12.34361708164215 ", "45.501602153542464", "12.343729734420776"],
+            ["45.501602153542464", "12.343729734420776", "45.50167359089988",  "12.343917489051817"],
+            ["45.50167359089988",  "12.343917489051817", "45.50177886684036",  "12.344056963920593"],
+            ["45.50150063714179",  "12.34214186668396 ", "45.50149311740111",  "12.341943383216856"],
+            ["45.50149311740111",  "12.341943383216856", "45.50149311740111",  "12.341868281364441"],
+            ["45.50149311740111",  "12.341868281364441", "45.501463038428405", "12.341744899749756"],
+            ["45.501463038428405", "12.341744899749756", "45.501421679814676", "12.34164297580719 "],
+            ["45.501421679814676", "12.34164297580719 ", "45.50136528165613",  "12.341508865356445"],
+            ["45.49857162216272",  "12.336707711219788", "45.498142973043684", "12.337351441383362"],
+            ["45.498142973043684", "12.337351441383362", "45.498124172569014", "12.337431907653809"],
+            ["45.498124172569014", "12.337431907653809", "45.49811289228121",  "12.337555289268494"],
+            ["45.49811289228121",  "12.337555289268494", "45.49813921294925",  "12.33765721321106 "],
+            ["45.49813921294925",  "12.33765721321106 ", "45.49871450447727",  "12.33838677406311 "],
+            ["45.498244495498355", "12.338268756866455", "45.49831969719862",  "12.338258028030396"],
+            ["45.49831969719862",  "12.338258028030396", "45.498372338329055", "12.338193655014038"],
+            ["45.498372338329055", "12.338193655014038", "45.49838361856487",  "12.33811855316162 "],
+            ["45.49838361856487",  "12.33811855316162 ", "45.498379858486516", "12.337989807128906"],
+            ["45.498379858486516", "12.337989807128906", "45.49833097744499",  "12.3378986120224  "],
+            ["45.49816929369764",  "12.336283922195435", "45.497729361151194", "12.336970567703245"],
+            ["45.497729361151194", "12.336970567703245", "45.49771808078429",  "12.337045669555664"],
+            ["45.49771808078429",  "12.337045669555664", "45.497725601029146", "12.337131500244139"],
+            ["45.497725601029146", "12.337131500244139", "45.49776320223832",  "12.337195873260498"],
+            ["45.49776320223832",  "12.337195873260498", "45.49815801341887",  "12.337673306465149"],
+            ["45.502857933880314", "12.343418598175049", "45.50609125134568",  "12.347688674926758"],
+            ["45.50601606002538",  "12.343976497650146", "45.50405353104642",  "12.346991300582886"],
+        
```

Comment:
I assume these empty lines are here to help you distinguish between different sections of the video map. You can instead define multiples separate sections in your video map.

Reply:
Cannot solve at this time, for now I'll remove spaces. When I have time I'll do it.

Reply:
Works for me 👍

Reply:
Thanks, I removed the spaces. Ready for review.